osu.Game/Screens/OnlinePlay/Components/StarRatingRangeDisplay.cs

```diff
@@ -85,6 +89,7 @@ private void updateRange(object sender, NotifyCollectionChangedEventArgs e)
 
             minDisplay.Current.Value = minDifficulty;
             maxDisplay.Current.Value = maxDifficulty;
+            maxDisplay.Alpha = Precision.AlmostEquals(minDifficulty.Stars, maxDifficulty.Stars) ? 0 : 1;
```

Comment:
This is using the default precision of `1e-7`, while star ratings are rounded to 2 decimal places - it may be the case that this will display ranges like [3.00*, 3.00*] if the underlying numbers are something like 2.997*-3.002*.

Setting the precision to `0.005` is not necessarily valid here either, though (consider ranges like 2.9901*-2.9951*). The way to go here would probably be to round both to 2 decimal places and compare that (probably directly at that point?)